tests/bootstrap-test-env.sh

```diff
@@ -32,7 +32,7 @@ else
 fi
 
 echo "Checking Vagrant installation..."
-vagrant_version=`vagrant --version | egrep -o "[0-9]+\.[0-9]+\.[0-9]+"`
+vagrant_version=`vagrant --version | egrep --colour=never -o "[0-9]+\.[0-9]+\.[0-9]+"`
```

Comment:
To avoid needing to modify every `egrep` command manually, consider using

`export GREP_OPTIONS='--color=never'`